blueprints/data-solutions/data-platform-minimal/02-processing.tf

```diff
@@ -0,0 +1,141 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# tfdoc:file:description Processing project and VPC.
+
+locals {
+  iam_processing = {
+    "roles/composer.admin"                            = [local.groups_iam.data-engineers]
+    "roles/composer.environmentAndStorageObjectAdmin" = [local.groups_iam.data-engineers]
+    "roles/composer.ServiceAgentV2Ext" = [
+      "serviceAccount:${module.processing-project.service_accounts.robots.composer}"
+    ]
+    "roles/composer.worker" = [
+      module.processing-sa-cmp-0.iam_email
+    ]
+    "roles/dataproc.editor" = [
+      module.processing-sa-cmp-0.iam_email
+    ]
+    "roles/dataproc.worker" = [
+      module.processing-sa-dp-0.iam_email
+    ]
+    "roles/iam.serviceAccountUser" = [
+      module.processing-sa-cmp-0.iam_email, local.groups_iam.data-engineers
+    ]
+    "roles/iap.httpsResourceAccessor"         = [local.groups_iam.data-engineers]
+    "roles/serviceusage.serviceUsageConsumer" = [local.groups_iam.data-engineers]
+    "roles/storage.admin" = [
+      module.processing-sa-cmp-0.iam_email,
+      "serviceAccount:${module.processing-project.service_accounts.robots.composer}",
+      local.groups_iam.data-engineers
+    ]
+  }
+  processing_subnet = (
+    local.use_shared_vpc
+    ? var.network_config.subnet_self_links.processingestration
+    : values(module.processing-vpc.0.subnet_self_links)[0]
+  )
+  processing_vpc = (
+    local.use_shared_vpc
+    ? var.network_config.network_self_link
+    : module.processing-vpc.0.self_link
+  )
+
+
+}
+
+module "processing-project" {
+  source          = "../../../modules/project"
+  parent          = var.project_config.parent
+  billing_account = var.project_config.billing_account_id
+  project_create  = var.project_config.billing_account_id != null
+  prefix          = var.project_config.billing_account_id == null ? null : var.prefix
+  name            = var.project_config.billing_account_id == null ? var.project_config.project_ids.processing : "${var.project_config.project_ids.processing}${local.project_suffix}"
```

Comment:
can you break long lines, non only here but in all the blueprint files? this for example should be

```hcl
name            = (
  var.project_config.billing_account_id == null
  ? var.project_config.project_ids.processing
  : "${var.project_config.project_ids.processing}${local.project_suffix}"
)
```